### ISSUE TITLE
Jetpack Plugins: Add infinite scroll paging

### ISSUE THREAD

**Issue by v18** [Automattic/wp-calypso#408]
Would be really nice to have. Gotta make sure we nail the other stuff first. This kind of thing is going to be necessary when we get to the plugin browser. Might be good to get it set up and tested for this, first.

**Comment by enejb** [comment]
We have this implemented in the plugin browser. Right now there is no way to retrieve a paginated plugin list form the Jetpack API side.

**Comment by v18** [comment]
@enejb - I noticed that there's infinite scroll in the plugin browser, but the URL doesn't reflect which page the user is on. Here's a gif showing that:

![no-page-numbers-when-scrolling-plugins](https://cloud.githubusercontent.com/assets/11487924/12646601/1634f1fe-c59f-11e5-9213-2972c1f030d0.gif)

As for the plugins view for a site, is there a way to paginate it without waiting on the Jetpack API? Not sure if it's possible to just get a list of plugins without all of their details right away, and then fill the details in later?
